tests/sentry/workflow_engine/handlers/condition/test_age_comparison_handler.py

```diff
@@ -0,0 +1,90 @@
+from datetime import datetime, timedelta, timezone
+
+from sentry.rules.age import AgeComparisonType
+from sentry.rules.filters.age_comparison import AgeComparisonFilter
+from sentry.testutils.helpers.datetime import freeze_time
+from sentry.workflow_engine.models.data_condition import Condition
+from sentry.workflow_engine.types import WorkflowJob
+from tests.sentry.workflow_engine.handlers.condition.test_base import ConditionTestCase
+
+
+class TestAgeComparisonCondition(ConditionTestCase):
+    condition = Condition.AGE_COMPARISON
+    rule_cls = AgeComparisonFilter
+    payload = {
+        "id": AgeComparisonFilter.id,
+        "comparison_type": AgeComparisonType.OLDER,
+        "value": "10",
+        "time": "hour",
+    }
+
+    def setup_group_event_and_job(self):
+        self.group_event = self.event.for_group(self.group)
+        self.job = WorkflowJob(
+            {
+                "event": self.group_event,
+            }
+        )
+
+    # @freeze_time(datetime.now().replace(hour=0, minute=0, second=0, microsecond=0))
```

Comment:
do you need this? you could also put the freeze time before the class definition to have it apply to every function in the class

Reply:
oops good catch ty